### DISCUSSION TITLE
On Vanish, show Logoff and remove from Online List (toggle to turn that on/off)

### DISCUSSION THREAD

**Discussion by kellyhamlin** [Zrips/CMI#3307] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###

One of the competitor plugins, VanishNoPacket has a feature where when you go to vanish, it shows you log off the game server and removes you from the active online list. This is helpful as some players behave differently when mods/admins are online.
 /v
fffizzz has logged off (or whatever the existing messaging is)
<tab> no longer in tablist

maybe a different command altogether rather than a toggle, like /vl  (vanish/logout)

**Comment by CloudeLecaw** [comment]
beeing silent is allready an option of cmi vanished
check the cmi vanishedit and the vanish default if yoi dont wanne use the vanish Editor.

if your triggering your logon messeage with eventCommands.yml instead of cmis default custom then you can also use a check for the vanishsymbol placeholder to trigger a message or not.
